api.py

```diff
@@ -275,7 +271,7 @@ def get(self, user_id):
             if convo_id not in messages:
                 messages[convo_id] = []
             message_obj = {'message': message,
-                           'message_id': message_id,
+                           'message_id': message_id[0],
```

Comment:
same here, change to `for message_id, in query: and get rid of accessing by index